src/mrinufft/trajectories/utils.py

```diff
@@ -173,69 +177,220 @@ class VDSpdf(StrEnum):
+
+    The default values for the parameters are set to typical values used in MRI systems.
+    These values can be adjusted based on the specific hardware being used for MRI
+    acquisition.
+
+    For convenience, we define several common hardware configurations,
+
+    """
+
+    gmax: float = 40 * SI.milli  # Maximum gradient amplitude in T/m
+    smax: float = 200  # T/m/s
+    n_coils: int = 32
+    dwell_time: float = 1 * SI.nano  # s
+    grad_raster_time: float = 5 * SI.micro  # s
+    field_strength: float = 3.0  # Tesla
+
+    @property
+    def raster_time(self) -> float:
+        """Alias of grad_raster_time."""
+        return self.grad_raster_time
+
+
+# fmt: off
+class SIEMENS:
+    """Common hardware configurations for Siemens MRI systems."""
+
+    TERRA          = Hardware(gmax=80*SI.milli, smax=200, field_strength=7)
+    TERRAX         = Hardware(gmax=200*SI.milli, smax=200, field_strength=7)
+    TERRAX_IMPULSE = Hardware(gmax=200*SI.milli, smax=900, field_strength=7)
+    PRISMA         = Hardware(gmax=40*SI.milli, smax=200, field_strength=3)
+    CIMA           = Hardware(gmax=200*SI.milli, smax=200, field_strength=3)
+    CIMAX          = Hardware(gmax=200*SI.milli, smax=200, field_strength=3)
+
+# fmt: on
+
+
+@dataclass(frozen=True)
+class Acquisition:
+    """
+    Acquisition configuration for MRI sequences.
+
+    Parameters
+    ----------
+    fov : tuple[float, float, float]
+        Field of View in meters (x, y, z).
+    img_size : tuple[int, int, int]
+        Image size in pixels (x, y, z).
+    hardware : Hardware
+        Hardware configuration for the acquisition.
+    gamma : Gammas, optional
+        Gyromagnetic ratio in Hz/T for the nucleus being imaged.
+        Defaults to `Gammas.HYDROGEN`.
+    oversampling : int, optional
+        Oversampling factor for the ADC. Defaults to 1.
+    norm_factor : float, optional
+        Normalization factor for the trajectory. Defaults to 0.5.
+
+    Attributes
+    ----------
+    default : ClassVar[Acquisition]
+        The default acquisition configuration used if none is specified.
+
+    Notes
+    -----
+    The `Acquisition` class encapsulates the parameters needed for MRI acquisition,
+    including the field of view, image size, hardware specifications, and gyromagnetic
+    ratio.
+
+    It is designed to be used in conjunction with the `Hardware` class, which defines
+    the hardware constraints such as maximum gradient amplitude and slew rate.
+    The `default` class variable holds the default acquisition configuration, which can
+    be set using the `set_default` method. This allows for easy access to a standard
+    acquisition configuration without needing to instantiate a new `Acquisition` object
+    each time.
+
+    """
+
+    default: ClassVar[Acquisition]
+
+    fov: tuple[float, float, float]  # Field of View in m
+    img_size: tuple[int, int, int]  # Image size in pixels
+    hardware: Hardware = SIEMENS.TERRA  # Hardware configuration
```

Comment:
I think having hardware inside Acquisition could over-complicate things. We dont always need a hardware config really, only when we specify or check for hardware constraints

Reply:
To me we can keep hardware and acquisition as 2 different dataclasses and not nest them.

Reply:
I disagree. An acquisition is always performed on some specific (or a generic preset) hardware. Having a single structure to pass by simplifies everything.

Reply:
I agree about simplification. But i still feel acquisition and hardware are fundamentally different things. Within this PR itself i see the same dataclass being sent even if we use only one of them. I'm ok with it for simplification but it's better to be mindful that we potentially make it harder for users to just give hardware config to check constraints etc...